.github/workflows/lint.yml

```diff
@@ -26,6 +26,8 @@ jobs:
       - name: Install dependencies
         run: |
           pip install black
+      - name: Check black version
+        run: black --version
```

Comment:
If we have the CICD running, then probably yes. For now, I just wanted to check what the current running version was so that I could match that version locally

Reply:
Cause if we have the CICD, we also have to deliberately switch the version if needed so that all changes are in 1 PR